src/app/profile/info-form/info-form.component.ts

```diff
@@ -1,33 +1,46 @@
-import { Component } from '@angular/core';
-import {MatSelectModule} from '@angular/material/select';
-import {MatInputModule} from '@angular/material/input';
-import {MatFormFieldModule} from '@angular/material/form-field';
-import {MatDatepickerModule} from '@angular/material/datepicker';
-import {MatNativeDateModule} from '@angular/material/core';
+import { Component, Input, OnInit, Output, EventEmitter, OnChanges, SimpleChanges } from '@angular/core';
+import { MatSelectModule } from '@angular/material/select';
+import { MatInputModule } from '@angular/material/input';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatNativeDateModule } from '@angular/material/core';
 import { FormControl, FormGroup, Validators } from '@angular/forms';
-
+import { Profile } from '../../shared/models/profile.model';
 
 @Component({
   selector: 'app-info-form',
   templateUrl: './info-form.component.html',
-  styleUrl: './info-form.component.css',
+  styleUrls: ['./info-form.component.css'],
   providers: [
     MatFormFieldModule,
     MatInputModule,
     MatSelectModule,
     MatDatepickerModule,
-    MatNativeDateModule
+    MatNativeDateModule,
   ],
 })
-export class InfoFormComponent {
+export class InfoFormComponent  {
+  @Output() profileChange = new EventEmitter<Profile>();
+  @Input() profile!: Profile;
```

Comment:
Add an output to tell if the form is valid.
```suggestion
  @Input() profile!: Profile;  
  
  @Input()
  valid: boolean = false;

  @Output() 
  validChange = new EventEmitter<boolean>();
```